build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -e
+
+if [ $(dotnet --info | grep "OS Platform" | grep -c Windows) -ne 0 ]
+then
+  OS=Windows
+else
+  OS=Linux
+fi
+
+FIND=/usr/bin/find
```

Comment:
Nit: As a general note in style, keep the non-exported (local) variables lowercase.